Symphony/Composer.swift

```diff
@@ -22,36 +22,43 @@ public protocol Composer: class {
     /// A reference to the currently presented composable.
     /// This is generally only necissary for memory management,
     /// and shouldn't need to be messed with.
-    var currentComposable: Composable? { get set }
```

Comment:
Sounds good, if it's non trivial and it was a "nice to have" lets punt till it becomes more of a need